src/app/components/rich-text-editor/prosemirror-editor-view.tsx

```diff
@@ -18,12 +18,6 @@ export class ProseMirrorEditorView extends React.Component<ProseMirrorEditorView
     this.props.onChange(tx);
   };
 
-  focus() {
-    if (this.editorView) {
-      this.editorView.focus();
-    }
-  }
-
   componentWillReceiveProps(nextProps: ProseMirrorEditorViewProps) {
```

Comment:
I get a warning when running the tests that 'componentWillReceiveProps' is deprecated, is it worth swapping this to an alternative?